openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -0,0 +1,167 @@
+"""
+Tests for course_overviews app.
+"""
+import datetime
+import ddt
+import itertools
+import time
+
+from django.utils import timezone
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from xmodule.modulestore import ModuleStoreEnum
+
+from lms.djangoapps.certificates.api import get_active_web_certificate
+from lms.djangoapps.courseware.courses import course_image_url
+
+from .models import CourseOverview
+
+
+@ddt.ddt
+class CourseOverviewTestCase(ModuleStoreTestCase):
+    """
+    Tests for CourseOverviewDescriptor model.
+    """
+
+    TODAY = timezone.now()
+    YESTERDAY = TODAY - datetime.timedelta(days=1)
+    TOMORROW = TODAY + datetime.timedelta(days=1)
+    NEXT_MONTH = TODAY + datetime.timedelta(days=30)
+
+    def check_course_overview_against_course(self, course):
```

Comment:
Please also test calling `start_datetime_text` with the `format_string` being "DATE_TIME".